csp_billing_adapter/adapter.py

```diff
@@ -25,23 +25,18 @@
 from csp_billing_adapter.config import Config
 from csp_billing_adapter.csp_cache import (
     add_usage_record,
-    cache_meter_record,
-    create_cache,
-    get_billing_dimensions,
-    get_billable_usage
+    create_cache
 )
 from csp_billing_adapter.csp_config import (
     create_csp_config,
 )
-from csp_billing_adapter.utils import get_now, date_to_string
+from csp_billing_adapter.utils import get_now, string_to_date
+from csp_billing_adapter.bill_utils import process_metering
 from csp_billing_adapter import hookspecs
 from csp_billing_adapter import csp_hookspecs
 from csp_billing_adapter import storage_hookspecs
 from csp_billing_adapter import (
     local_csp,
-    product_api,
-    memory_cache,
-    memory_csp_config
 )
 
 default_config_path = '/etc/csp_billing_adapter/config.yaml'
```

Comment:
should this be all caps ? I probably did this lower case and did not consider it should be caps

Reply:
Yeah since this should be considered a constant... I can fix it here if you want?

Reply:
please fix , thank you

Reply:
Done